### PR TITLE
Invites an existing user to join an Enterprise

### PR DESCRIPTION
**Pull request by stanislav-dvorscak**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @stanislav-dvorscak, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by stanislav-dvorscak** [comment]
CLA signed

**Comment by boxcla** [comment]
Verified that @stanislav-dvorscak has just signed the CLA. Thanks, and we look forward to your contribution.

File: src/main/java/com/box/sdk/BoxUserInvite.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+     */
+    private Date modifiedAt;
+
+    /**
+     * Constructs an empty Invite.
+     *
+     * @param api the API connection to be used by the resource.
+     */
+    public BoxUserInvite(BoxAPIConnection api) {
+        this.api = api;
```
Comment (dmaynard) [inline src/main/java/com/box/sdk/BoxUserInvite.java]: call super() to properly initialize the BoxJSONObject;

File: src/main/java/com/box/sdk/BoxUserInvite.java [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+     * Constructs an Invite from a JSON string.
+     *
+     * @param api  the API connection to be used by the resource.
+     * @param json the json encoded user invite.
+     */
+    public BoxUserInvite(BoxAPIConnection api, String json) {
+        super(json);
+        this.api = api;
+    }
+
```
Comment (dmaynard) [inline src/main/java/com/box/sdk/BoxUserInvite.java]: Add Java Docs
    /**
     * Constructs an Invite from a JSON string.
     *
     * @param api  the API connection to be used by the resource.
     * @param json the JOSNString encoded user invite.
     */

**Comment by gleb-urvanov** [comment]
David, thank you for the review! Requested changes are made now.

**Pull request closed without merging**